docs/src/pages/ui/getting-started/installation/installation.vue.mdx

```diff
@@ -3,7 +3,7 @@ import BlockSwitcher from 'amplify-docs/src/components/BlockSwitcher';
 
 [![version](https://img.shields.io/npm/v/@aws-amplify/ui-vue/next.svg)](https://www.npmjs.com/package/@aws-amplify/ui-angular/v/next)
 
-Install `@aws-amplify/ui-vue@next` with [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/):
+For Vue 3 install `@aws-amplify/ui-vue@next` with [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/):
```

Comment:
I wanted to explicitly call out this package is only for Vue 3

Reply:
Makes sense!